### PR TITLE
Rebranding for CLI.md file

### PR DESCRIPTION
**Pull request by popenta**

### CODE REVIEW

File: .github/workflows/build-ubuntu.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-name: Install erdpy using erdpy-up (Ubuntu)
```
Comment (andreibancioiu) [inline .github/workflows/build-ubuntu.yml:1]: Without "using erdpy-up", but can be removed in another PR.

File: erdpy/cli.py [hunk 2]
```diff
@@ -57,25 +57,25 @@ def _do_main(cli_args: List[str]):
...
+https://docs.multiversx.com/sdk-and-tools/mxpy.
         """,
         formatter_class=argparse.RawDescriptionHelpFormatter
     )
 
     parser._positionals.title = "COMMAND GROUPS"
     parser._optionals.title = "TOP-LEVEL OPTIONS"
 
-    parser.add_argument("-v", "--version", action="version", version=f"erdpy {__version__}")
+    parser.add_argument("-v", "--version", action="version", version=f"mxpy {__version__}")
```
Comment (andreibancioiu) [inline erdpy/cli.py:78]: I think we can say: `MultiversX Python CLI (mxpy) ...`.

File: .github/workflows/build-macos.yml [hunk 3]
```diff
@@ -1,4 +1,4 @@
-name: Install erdpy using erdpy-up (MacOS)
+name: Install mxpy using erdpy-up (MacOS)
```
Comment (ssd04) [inline .github/workflows/build-macos.yml:1]: so `erdpy-up` will remain?

File: erdpy/cli.py (hunk 2 above)
Comment (andreibancioiu) [reply erdpy/cli.py:78]: Can be done in the next PR.

File: .github/workflows/install.yml [hunk 4]
```diff
@@ -1,7 +1,7 @@
 # This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
 # For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
 
-name: erdpy-build
+name: mxpy-build
```
Comment (andreibancioiu) [inline .github/workflows/install.yml:4]: In the workflows files, we can simply remove `mxpy-` as well, and only leave `build:`  etc.

File: .github/workflows/build-ubuntu.yml [hunk 5]
```diff
@@ -9,7 +9,7 @@ env:
  BRANCH_NAME: ${{ github.head_ref || github.ref_name }}
 
 jobs:
-  install-erdpy:
+  install-mxpy:
```
Comment (andreibancioiu) [inline .github/workflows/build-ubuntu.yml:12]: Can be just `install:`.

File: .github/workflows/install.yml [hunk 6]
```diff
@@ -28,8 +28,8 @@ permissions:
   statuses: none
 
 jobs:
-  build-erdpy:
-    name: Build and Test erdpy for ${{ matrix.os }}, python ${{ matrix.python-version }}
+  build-mxpy:
```
Comment (andreibancioiu) [inline .github/workflows/install.yml:31]: Can be `build:`.

File: .github/workflows/build-macos.yml (hunk 3 above)
Comment (andreibancioiu) [inline .github/workflows/build-macos.yml:1]: Can be `Install (MacOS)`.

File: .github/workflows/build-macos.yml [hunk 7]
```diff
@@ -9,7 +9,7 @@ env:
  BRANCH_NAME: ${{ github.head_ref || github.ref_name }}
 
 jobs:
-  install-erdpy:
+  install-mxpy:
```
Comment (andreibancioiu) [inline .github/workflows/build-macos.yml:12]: Can be `install:`.

**Review by andreibancioiu — DISMISSED** [review]

File: .github/workflows/build-macos.yml (hunk 3 above)
Comment (popenta) [reply .github/workflows/build-macos.yml:1]: Deleted it.

**Review by andreibancioiu — APPROVED** [review]

**Review by ssd04 — APPROVED** [review]
